pkg/scheduler/factory.go

```diff
@@ -323,16 +323,12 @@ func (c *Configurator) createFromConfig(policy schedulerapi.Policy) (*Scheduler,
 	}
 	for i := range c.profiles {
 		prof := &c.profiles[i]
-		if prof.Plugins != nil {
-			return nil, errors.New("using Plugins and Policy simultaneously is not supported")
-		}
+		// Plugins are empty when using Policy.
 		prof.Plugins = &schedulerapi.Plugins{}
 		prof.Plugins.Append(&defPlugins)
 
-		if len(prof.PluginConfig) != 0 {
-			return nil, errors.New("using PluginConfig and Policy simultaneously is not supported")
```

Comment:
is this error not possible any more? silently stomping `prof.PluginConfig` seems like it could be problematic

Reply:
Yes. This was only necessary for v1alpha1, which is now removed.

v1alpha1 allowed having Policy and plugin configuration at the same time. Now, the only way to use Policy is via command line flag, which then implies that you don't have a component config file.

Reply:
Added comment in `SetDefaults_InterPodAffinityArgs`